README.md

```diff
@@ -42,3 +42,15 @@ in this repository's root directory.
 
 After running DocC, open the link that `docc` outputs
 to display a local preview in your browser.
+
+> Note:
+>
+> If you installed DocC by downloading a toolchain from Swift.org,
+> `docc` is located in `usr/bin/`,
```

Comment:
Nit:

Should this be `/usr/bin` or `usr/bin/`.

IMO, `/usr/bin` is a more common express for relative path.

```suggestion
> `docc` is located in `/usr/bin/`,
```

Reply:
The leading `/` is for an absolute path.  Relative paths start with `./` or `../` or the name of a file or directory.

I could go either way on the trailing slash.  I included it to emphasize the fact that `bin/` is a directory, but it's also correct to omit it.

Reply:
```
type id:                    com.apple.dt.document.toolchain (0xeb04)
bundle:                     Xcode (0xed8)
uti:                        com.apple.dt.document.toolchain
localizedDescription:       "en" = ?, "LSDefaultLocalizedValue" = "Xcode Toolchain"
flags:                      active  apple-internal  exported  trusted (0000000000000055)
icons:                      2 values (45420 (0xb16c))
                            {
                                UTTypeIconBadgeName = "hammer-badge";
                                UTTypeIconText = toolchain;
                            }
conforms to:                public.composite-content, com.apple.package
tags:                       .xctoolchain
```

`xctoolchain` conforms to a Uniform Type Identifiers kind of `com.apple.package` which IMO is some kind of small file system. So we should use a absolute path `/usr/bin/` for this small file system. 

> "A single file system object can be a package and a bundle."
> https://developer.apple.com/documentation/uniformtypeidentifiers/uttype/3551543-package

Reply:
I don't have an opinion here either way; if I were speaking aloud to someone trying to convey where the DocC binary lived, I'd say something like "it ships in the Swift toolchain under user bin" and trust that they'd understand the ambiguity.

Reply:
Actually I'm fine with the current state. (This is just a small suggestion not a "request change")

Is there any other block issues preventing this being merged? @amartini51 @krilnon

Reply:
@Kyle-Ye Thanks for taking the time to make sure this detail is correct. Looking at [Bundle Programming Guide](https://developer.apple.com/library/archive/documentation/CoreFoundation/Conceptual/CFBundles/AboutBundles/AboutBundles.html#//apple_ref/doc/uid/10000123i-CH100-SW1), Apple's docs define a package as "any directory that the Finder presents to the user as if it were a single file."  So I prefer to omit the leading `/` and use a relative path.

> A single file system object can be a package and a bundle.

This doesn't mean a package is a kind of file system.  The same directory in the file system can be both a package and a bundle, at the same time.  "Bundle Programming Guide" talks about this in more detail, which maybe should be added to the page you linked:

> The reason bundles and packages are sometimes considered to be interchangeable is that many types of bundles are also packages. For example, applications and loadable bundles are packages because they are usually treated as opaque directories by the system. However, not all bundles are packages and vice versa.

Reply:
Got it. I'll check the source you provide. And I think we are ready for a merge.